astropy/coordinates/funcs.py

```diff
@@ -149,11 +152,23 @@ def get_sun(time):
     from .builtin_frames import GCRS
 
     earth_pv_helio, earth_pv_bary = erfa.epv00(time.jd1, time.jd2)
-    x = -earth_pv_helio[..., 0, 0] * u.AU
-    y = -earth_pv_helio[..., 0, 1] * u.AU
-    z = -earth_pv_helio[..., 0, 2] * u.AU
+
+    # We have to manually do abberation because we're outputting directly into
```

Comment:
Actually, @cdeil, after trying to do this I realized I couldn't really figure out a way to make it any simpler to actually invoke than the already existing usage, because in all other cases you would want to specify the earth->something else axis.  Here it's only simpler because the "something else" is the sun itself.  So I don't really see a way to generalize it more than what's in the existing ERFA ``ab`` function. (except for the ``invlorentz`` bit... not sure why ERFA decided that should be stored separately... But that's a one-liner and it doesn't seem to me like it justifies a helper at this point)